mrbgems/mruby-fiber/src/fiber.c

```diff
@@ -259,6 +259,9 @@ fiber_switch(mrb_state *mrb, mrb_value self, mrb_int len, const mrb_value *a, mr
  *  to the next <code>Fiber.yield</code> statement inside the fiber's block
  *  or to the block value if it runs to completion without any
  *  <code>Fiber.yield</code>
+ *
+ *  This method cannot be called from C.
```

Comment:
Is this comment needed/useful? It's a static function that never be able to be called from outside.

Reply:
This will be a comment on a Ruby method.
So I believe it will be useful.

Reply:
Probably better to mention `mrb_funcall` in the comment.

Reply:
Indeed it is. I will update it.